deployment/CP02PMUO_Deploy.csv

```diff
@@ -134,3 +134,12 @@ AR44,,,,CP02PMUO-WFP01-02-DOFSTK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015
 AR44,,,,CP02PMUO-WFP01-03-CTDPFK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,CGVEH-CWFPCP-10304,CGINS-CTDPFK-00127,39.942883,-70.770933,,419,443,
 AR44,,,,CP02PMUO-WFP01-04-FLORTK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,CGVEH-CWFPCP-10304,CGINS-FLORTK-01100,39.942883,-70.770933,,419,443,
 AR44,,,,CP02PMUO-WFP01-05-PARADK000,15,1,2020-06-09T13:57:00,,CGMCP-02PMUO-00015,CGVEH-CWFPCP-10304,CGINS-PARADK-20582,39.942883,-70.770933,,419,443,
+AR48,,,,CP02PMUO-RII01-02-ADCPSL010,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,,CGINS-ADCPSL-22846,39,-70,,,,
+AR48,,,,CP02PMUO-SBS01-00-STCENG000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,,CGCON-BSTC01-00005,39,-70,,0,,
+AR48,,,,CP02PMUO-SBS01-01-MOPAK0000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,,CGINS-MOPAK0-11358,39,-70,,0,,
+AR48,,,,CP02PMUO-WFP01-00-WFPENG000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGCON-CWFPCP-10301,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-01-VEL3DK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-VEL3DK-00035,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-02-DOFSTK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-DOFSTK-02708,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-03-CTDPFK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-CTDPFK-00124,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-04-FLORTK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-FLORTK-01117,39,-70,,,,
+AR48,,,,CP02PMUO-WFP01-05-PARADK000,16,1,2020-10-28T00:00:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-PARADK-20610,39,-70,,,,
```

Comment:
Looks good. Confirmed SNs with config sheet

Reply:
Looks good.  Compared to Config Sheet, WFP QCT 3310-00100-00127, and WFP instrument cal docs.